### ISSUE TITLE
muVT_sim docs out of date

### ISSUE THREAD

**Issue by SimonEnsemble** [SimonEnsemble/PorousMaterials.jl#233]
`kwargs` for the muVT_sim out of date

https://github.com/SimonEnsemble/PorousMaterials.jl/blob/master/src/gcmc.jl#L181-L182

does not match the docs https://simonensemble.github.io/PorousMaterials.jl/dev/gcmc/ `n_sample_cycles`.

also, if we want to run with one cycle, this is impossible to do with current setup. think this can be done via `n_cycles_per_volume` being a Float. and making min of 1.

**Comment by SimonEnsemble** [comment]
```
IOError: mkdir("/Users/cokes/.julia/pluto_notebooks/data/simulations"; mode=0o777): no such file or directory (ENOENT)```

create directory for snapshots file if it doesn't exist

**Comment by eahenle** [comment]
resolved in v0.5
